src/house.cpp

```diff
@@ -439,7 +442,7 @@ void AccessList::parseList(const std::string& list)
 				addGuildRank(line.substr(0, at_pos - 1), line.substr(at_pos + 1));
 			}
 		} else if (line.find("!") != std::string::npos || line.find("*") != std::string::npos || line.find("?") != std::string::npos) {
-			addExpression(line);
+			continue; // regexp no longer supported
 		} else {
 			addPlayer(line);
```

Comment:
what are you think about check if the line contains only valid characters? [A-Za-z- ']

Reply:
I don't think it matters here, we only match it against the character name with an exact match.